libs/features/mas/web-components/src/variants/ccd-suggested.css.js

```diff
@@ -0,0 +1,28 @@
+export const CSS = `
+:root {
+  --consonant-merch-card-ccd-suggested-width: 304px;
+  --consonant-merch-card-ccd-suggested-height: 205px;
+  --consonant-merch-card-ccd-suggested-background-img-size: 119px;
+}
+
+merch-card[variant="ccd-suggested"] [slot='detail-m'] {
+  color: var(--merch-color-grey-60);
+}
+
+merch-card[variant="ccd-suggested"] [slot='icons'] {
+    flex-flow: wrap;
+    place-self: flex-start;
+}
+
+merch-card[variant="ccd-suggested"] [slot='heading-xs'] {
+  font-size: var(--consonant-merch-card-heading-xxs-font-size);
+  line-height: var(--consonant-merch-card-heading-xxs-line-height);
+
+}
```

Comment:
this should be part of the card CSS, rather than the document's

Reply:
There is a problem with doing what you suggest: Global styles override VariantLayout styles. I suggest we discuss how to proceed with this in our tech call but it wouldn't hurt to leave it as is in the meantime.
![Screenshot 2024-10-01 at 1 26 49 PM](https://github.com/user-attachments/assets/54b0be61-cc11-46cd-9776-843f06d44962)